lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -60,7 +60,8 @@
  % endif
 <article class="course${mode_class}">
   <% course_target = reverse('info', args=[unicode(course_overview.id)]) %>
-  <section class="details">
+  <section class="details" aria-labelledby="details-heading">
+      <h2 class="hd hd-2 sr" id="details-heading">${_('Course details')}</h2>
```

Comment:
I've made this unique. The a11y test already checks for `duplicate_id`'s, I just don't think the test course it uses has more than one.